planner/core/point_get_plan.go

```diff
@@ -166,23 +168,30 @@ func (p *PointGetPlan) AccessObject(normalized bool) string {
 
 // OperatorInfo implements dataAccesser interface.
 func (p *PointGetPlan) OperatorInfo(normalized bool) string {
-	buffer := bytes.NewBufferString("")
+	var buffer strings.Builder
+	if p.Handle == nil && !p.Lock {
+		return ""
+	}
 	if p.Handle != nil {
 		if normalized {
-			fmt.Fprintf(buffer, "handle:?, ")
+			// fmt.Fprintf(buffer, "handle:?, ")
+			buffer.WriteString("handle:?")
 		} else {
 			if p.UnsignedHandle {
-				fmt.Fprintf(buffer, "handle:%d, ", uint64(p.Handle.IntValue()))
+				// fmt.Fprintf(buffer, "handle:%d, ", uint64(p.Handle.IntValue()))
+				buffer.WriteString("handle:" + strconv.FormatUint(uint64(p.Handle.IntValue()), 10))
 			} else {
-				fmt.Fprintf(buffer, "handle:%s, ", p.Handle)
+				// fmt.Fprintf(buffer, "handle:%s, ", p.Handle)
+				buffer.WriteString("handle:" + p.Handle.String())
```

Comment:
There is string concat here. I guess simply using `buffer.WriteString` × 2 will be more efficient.

Reply:
ok, sorry I forgot deleting old code...